stacks/op-scim-bridge/deploy-local.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+set -e
+
+ROOT_DIR=$(git rev-parse --show-toplevel)
+
+# create namespace
+cat <<EOF | kubectl apply -f -
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: op-scim-bridge
+EOF
+
+# set kubectl namespace
+# kubectl config set-context --current --namespace=op-scim-bridge
```

Comment:
Commenting this out results in op-scim-bridge being installed into whatever namespace the installer happens to be in (`default` in my case). Mind uncommenting it?

Reply:
Ah, the command generated an error for me, and I thought it only was used for my testing. Will uncomment.